src/managed/Microsoft.DotNet.Build.Bundle/Program.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DotNet.Build.Bundle
+{
+    /// <summary>
+    ///  The main driver for Bundle and Extract operations.
+    /// </summary>
+    public static class Program
+    {
+        enum RunMode
+        {
+            Help,
+            Bundle,
+            Extract
+        };
+
+        static RunMode Mode = RunMode.Bundle;
+
+        // Common Options:
+        static bool Verbose = false;
+        static string OutputDir;
+
+        // Bundle options:
+        static bool EmbedPDBs = false;
+        static string HostName;
+        static string SourceDir;
+
+        // Extract options:
+        static string BundleToExtract;
+
+        static void Usage()
+        {
+            Console.WriteLine($".NET Core Bundler (version {Bundler.Version})");
+            Console.WriteLine("Usage: bundle <options>");
+            Console.WriteLine("");
+            Console.WriteLine("Bundle options:");
+            Console.WriteLine("  --source <PATH>    Directory containing files to bundle (required).");
+            Console.WriteLine("  --apphost <NAME>   Application host within source directory (required).");
+            Console.WriteLine("  --pdb              Embed PDB files.");
+            Console.WriteLine("");
+            Console.WriteLine("Extract options:");
+            Console.WriteLine("  --extract <PATH>   Extract files from the specified bundle.");
+            Console.WriteLine("");
+            Console.WriteLine("Common options:");
+            Console.WriteLine("  -o|--output <PATH> Output directory (default: current).");
+            Console.WriteLine("  -d|--diagnostics   Enable diagnostic output.");
+            Console.WriteLine("  -?|-h|--help       Display usage information.");
+            Console.WriteLine("");
+            Console.WriteLine("Examples:");
+            Console.WriteLine("Bundle:  bundle --source <publish-dir> --apphost <host-exe> -o <output-dir>");
+            Console.WriteLine("Extract: bundle --extract <bundle-exe> -o <output-dir>");
+        }
+
+        public static void Log(string fmt, params object[] args)
+        {
+            if (Verbose)
+            {
+                Console.WriteLine("LOG: " + fmt, args);
+            }
+        }
+
+        static void Fail(string type, string message)
+        {
+            Console.Error.WriteLine($"{type}: {message}");
+        }
+
+        static void ParseArgs(string[] args)
```

Comment:
There are [command-line](https://github.com/dotnet/command-line-api) can use.

Reply:
@lindexi  this PR was merged on Feb 7th. The code that you've left comments on was removed later due to refactoring.